### ISSUE TITLE
Implement `metric` parameter for cross_validation_across_time

### ISSUE THREAD

**Issue by skjerns** [skjerns/meg_utils#1]
it would be nice to have a parameter with which we can use a different scoring algorithm for the function.

The parameter should accept either a string keyword or alternatively a function. if it's a string, it should take a function from https://scikit-learn.org/stable/modules/model_evaluation.html with the corresponding name

for the top-N you can use https://scikit-learn.org/stable/modules/generated/sklearn.metrics.top_k_accuracy_score.html

so you can call `cross_validation_across_time(..., metric='top-3')`

the default should be `metric='accuracy'`

Best,
Simon
